test/artefacts/overseas-passports/falkland-islands/replacing/adult.txt

```diff
@@ -17,7 +17,9 @@ Don’t book travel until you have a valid passport. If you need to travel more
 
 ## Cost
 
-You’ll have to pay a fee for your passport and a courier fee of £9.70. The courier fee pays for your passport to be sent back to you securely.
+You’ll have to pay a fee for your passport and a courier fee of £9.70.
+
+The courier fee pays for your passport to be sent back to you securely.
```

Comment:
Minor: I assume `falkland-islands` and `the-occupied-palestinian-territories` are the only regression test artefacts to change, because they are the only "IPS 1" countries. However, since I have no idea what "IPS 1" means, this wasn't at all obvious. It might be worth explaining in the commit note.